modules/exploits/multi/http/gitlab_file_read_rce.rb

```diff
@@ -147,7 +146,8 @@ def create_project(user:)
           'project[namespace_id]' => (user['id']).to_s,
           'project[path]' => project_name,
           'project[description]' => Rex::Text.rand_text_alphanumeric(8),
-          'project[visibility_level]' => '0'
+          'project[visibility_level]' => '0',
+          'keep_cookies' => true
```

Comment:
Just confirming if this in the right place? 👀

Reply:
Nope, good catch